Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java

```diff
@@ -323,62 +325,72 @@ public void stopWebApp(String sid, String appid) throws IOException {
     }
 
     /**
-     * List Web Apps by Subscription ID.
+     * List all the Web Apps on Windows in selected subscriptions.
      */
-    public List<ResourceEx<WebApp>> listWebAppsOnWindowsBySubscriptionId(String sid, boolean force) {
-        if (!force && subscriptionIdToWebAppsOnWindowsMap.containsKey(sid)) {
-            return subscriptionIdToWebAppsOnWindowsMap.get(sid);
-        }
-        List<ResourceEx<WebApp>> webAppList = new ArrayList<>();
-        try {
-            Azure azure = AuthMethodManager.getInstance().getAzureClient(sid);
-            for (WebApp webApp : azure.webApps().list()) {
-                if (webApp.operatingSystem().equals(OperatingSystem.WINDOWS)) {
-                    webAppList.add(new ResourceEx<>(webApp, sid));
-                }
-            }
-            subscriptionIdToWebAppsOnWindowsMap.put(sid, webAppList);
-        } catch (IOException e) {
-            e.printStackTrace();
+    public List<ResourceEx<WebApp>> listAllWebAppsOnWindows(boolean force) {
+        final List<ResourceEx<WebApp>> webAppList = new ArrayList<>();
```

Comment:
There are different names "webApps" and "webAppList" in similar statements. Please be consistent.